test/functional/wallet_multiwallet.py

```diff
@@ -141,7 +141,7 @@ def wallet_file(name):
 
         # should raise rpc error if wallet path can't be created
         err_code = -4 if self.options.descriptors else -1
-        assert_raises_rpc_error(err_code, "boost::filesystem::create_directory:", self.nodes[0].createwallet, "w8/bad")
+        assert_raises_rpc_error(err_code, "filesystem error:", self.nodes[0].createwallet, "w8/bad")
```

Comment:
1c655826850235fbf13dc90daf66e2b3facdc9ba

MSVC implementation of `std::filesystem` has different error messages:

```suggestion
        assert_raises_rpc_error(err_code, "filesystem error:" if sys.platform != 'win32' else "create_directories:", self.nodes[0].createwallet, "w8/bad")
```

(and `import sys`, of course).

Reply:
Wouldn't it make more sense to use EXEEXT to avoid issues when the tests are run in wine or so?